src/c/weaver/weave/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_library(WeavePass MODULE
+    # List your source files here.
+    perfflow_weave.cpp
+)
+
+# Use C++11 to compile our pass (i.e., supply -std=c++11).
```

Comment:
The other cmake file sets the `CMAKE_CXX_STANDARD` to `14`, but this comment mentions that these targets are supported in `11`.  Should we loosen the `CMAKE_CXX_STANDARD`?

Reply:
Our pass is C++ 11 but the overall LLVM Pass header files requires C++14. I will see if we say stay at C++14 though.